stonesoup/functions/__init__.py

```diff
@@ -509,6 +509,47 @@ def rotz(theta):
                      [zero, zero, one]])
 
 
+def gm_sample(means, covars, size, weights=None):
+    """Sample from a mixture of multi-variate Gaussians
+
```

Comment:
You might wish to explain how this is done, as for non-statisticians it might not be obvious. (First do a multinomial sample from the weight distribution to get the number of times you should sample from each component, then sample from those components that many times.. I think.)